projects/demo-integrations/cypress/support/commands.ts

```diff
@@ -24,7 +24,7 @@ declare global {
 
 Cypress.Commands.add('getByAutomationId', id => cy.get(`[automation-id=${id}]`));
 Cypress.Commands.add('findByAutomationId', {prevSubject: true}, (subject: any, id) =>
-    subject.find(`[automation-id=${id}]`),
+    cy.wrap(subject).find(`[automation-id=${id}]`),
```

Comment:
`subject` is just `jQuery`-element which has method `.find()`. 
**This method doesn't have [retry-ability](https://docs.cypress.io/guides/core-concepts/retry-ability#Commands-vs-assertions)!**
___
`cy.wrap(subject)` is a `$Chainer`-element (cypress built-in implementation) which **also** has method `.find()`.
**This method has [retry-ability](https://docs.cypress.io/guides/core-concepts/retry-ability#Commands-vs-assertions)!**

Reply:
add a comment to the code